storagenode/storagenodedb/infodb.go

```diff
@@ -269,6 +272,31 @@ func (db *InfoDB) Migration() *migrate.Migration {
 					`CREATE INDEX idx_pieceinfo_expiration ON pieceinfo(piece_expiration) WHERE piece_expiration IS NOT NULL`,
 				},
 			},
+			{
+				Description: "Clear Tables from Alpha data",
+				Version:     11,
+				Action: migrate.SQL{
+					`DELETE FROM pieceinfo`,
+					`DELETE FROM used_serial`,
+					`DELETE FROM order_archive`,
+				},
+			},
+			{
+				Description: "Free Storagenodes from trash data",
+				Version:     12,
+				Action: migrate.Func(func(log *zap.Logger, mgdb migrate.DB, tx *sql.Tx) error {
```

Comment:
Uh oh, the memory db has no location but still runs migrations. Doesn’t this remove stuff under `/blob`, etc.?

Reply:
I add a check for empty location path, that should fix that.